build-and-test.ps1

```diff
@@ -1,26 +1,30 @@
 #!/usr/bin/env pwsh
 [cmdletbinding()]
 param(
-    [ValidateSet('default', 'samples')]
-    [string]$ManifestType = 'default',
+    [ValidateSet('*', 'runtime-sdk', 'aspnet', 'wcf')]
+    [string]$ImageTypeFilter = '*',
```

Comment:
It feels more natural and efficient to me if this were using the `repo` filtering capabilities of ImageBuilder.  I also think renaming this parameter in terms of repo would be more natural and meaningful to users.  You could also easily support multiples by making this an array.

Reply:
I can rename it to `RepoFilter` but I can't make use of the `repo` option in ImageBuilder since that is limited to a single value.  So I'd still have to use the `path` option.